2025/program.html

```diff
@@ -46,7 +46,194 @@ <h1>SysTEX '25</h1>
 
   <div class="container">
     <h3>Program</h3>
-    Coming soon!
+
+
+    <div class="table-responsive-sm">
+      <table class="table table-hover table-condensed">
+        <tbody>
+          <tr>
+            <td>09:45-10:00</td>
+            <td>Opening and Welcome
+            </td>
+          </tr>
+          <tr>
+            <td>10:00-11:00</td>
+            <td><b>Keynote:</b> <i>Coming Soon</i> </td>
+          </tr>
+          <tr>
+            <td>11:00-11:30</td>
+            <td>Coffee Break</td>
+          </tr>
+          <tr>
+            <td>11:30-13:05</td>
+            <td><b>Session 1: Pot Pourri</b></td>
+          </tr>
+          <tr>
+            <td></td>
+            <td>
+              11:30:
+              <!--<a href="papers/systex24-final12.pdf"><i class="bi bi-file-earmark-pdf"></i>-->
+              An Early Experience with Confidential Computing Architecture for On-Device Model Protection
+              <!--</a>-->
+              <!--<a href="slides/slides-12.pdf">[ <i class="bi bi-images"></i> Slides ]</a>-->
+              <br>
+              <i> Sina Abdollahi (Imperial College London), Mohammad Maheri (Imperial College London), Sandra Siby (New York University Abu Dhabi), Marios Kogias (Imperial College London), Hamed Haddadi (Imperial College London & Brave Software)</i>
+              <br>
+            </td>
+          </tr>
+          <tr>
+            <td></td>
+            <td>11:55:
+              <!--<a href="papers/systex24-final72.pdf"><i class="bi bi-file-earmark-pdf"></i>-->
+              Proving Attributes about Confidential Compute Services with Validation and Endorsement Services
+              <!--</a>-->
+              <!--<a href="slides/slides-12.pdf">[ <i class="bi bi-images"></i> Slides ]</a>-->
+              <br>
+              <i> Anjo Vahldiek-Oberwagner (Intel Labs), Marcela S. Melara (Intel Labs)</i>
+              <br>
+              <div class="container">
+                <a href="cfp.html" role="button" class="btn btn-default btn-xs" style="font-variant-caps: small-caps;">
```

Comment:
I would not make this links to the CFP. There are 2 more instances in this document.